spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -664,6 +664,16 @@ Access logging for undertow can be configured in a similar fashion
 Logs are stored in a `logs` directory relative to the working directory of the
 application. This can be customized via `server.undertow.accesslog.directory`.
 
+Access logging for jetty can be configured in a similar fashion
+
+[source,properties,indent=0,subs="verbatim,quotes,attributes"]
+----
+	server.jetty.accesslog.enabled=true
+	server.jetty.accesslog.filename=
+----
+
+If no filename logs will go to System.err.
+For more details please refer to https://www.eclipse.org/jetty/documentation/9.3.x/configuring-jetty-request-logs.html.
```

Comment:
Ah you've answered my question above. That information should be set in the documentation of the `filename` property.